contentgrid-spring-data-rest/src/main/java/com/contentgrid/spring/data/rest/webmvc/DefaultDomainTypeToHalFormsPayloadMetadataConverter.java

```diff
@@ -202,4 +187,43 @@ public Optional<String> getPattern() {
         }
     }
 
+
+    @AllArgsConstructor
+    @RequiredArgsConstructor
+    static class ClassnameI18nedPayloadMetadata implements InputPayloadMetadata {
+        private final Class<?> domainType;
+        private final Collection<PropertyMetadata> properties;
+        @With
+        private List<MediaType> mediaTypes = List.of(MediaType.APPLICATION_JSON);
```

Comment:
Why is this a default mediatype?

Is that also what is set in spring-data-rest as default?